src/app/grid/grid-sample-selection-template-numbers/grid-sample-selection-template-numbers.component.scss

```diff
@@ -0,0 +1,43 @@
+.grid__wrapper {
+    margin: 0 16px;
+}
+
+.header-selector {
+    width: 70px;
+    height: 60px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.header-image {
+    width: 40px;
+    height: 40px;
+}
+
+.row-selector {
+    width: 70px;
+    display: flex;
+    justify-content: center;
```

Comment:
Yeah, those items don't really fit inside a box 50 box with internal padding where the number is 40. The checkbox is way off:
![image](https://user-images.githubusercontent.com/3198469/64632234-60775580-d401-11e9-9a3c-0335ae89fe4c.png)
You can remove the extra div (put the index span and checkbox as siblings) and go:
```suggestion
    justify-content: space-evenly;
```
With removed padding it looks centered and quite even on all pages.